packages/core/src/test/awsService/appBuilder/serverlessLand/wizard.test.ts

```diff
@@ -0,0 +1,132 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import * as vscode from 'vscode'
+import path from 'path'
+import { PrompterTester } from '../../../../test/shared/wizards/prompterTester'
+import { describe } from 'mocha'
+import { ProjectMetadata } from '../../../../awsService/appBuilder/serverlessLand/metadataManager'
+import * as nodefs from 'fs' // eslint-disable-line no-restricted-imports
+import fs from '../../../../shared/fs/fs'
+import globals from '../../../../shared/extensionGlobals'
+import { AppBuilderRootNode } from '../../../../awsService/appBuilder/explorer/nodes/rootNode'
+import * as sinon from 'sinon'
+import { AppNode } from '../../../../awsService/appBuilder/explorer/nodes/appNode'
+import { ResourceNode } from '../../../../awsService/appBuilder/explorer/nodes/resourceNode'
+import { getTestWindow } from '../../../../test/shared/vscode/window'
+
+describe('CreateWizard', async () => {
+    const metadataPath = globals.context.asAbsolutePath(path.join('dist', 'src', 'serverlessLand', 'metadata.json'))
+    const metadataContent = nodefs.readFileSync(metadataPath, { encoding: 'utf-8' })
+    const parseMetadata = JSON.parse(metadataContent) as ProjectMetadata
+    const workspaceFolder = vscode.workspace.workspaceFolders![0]
+    const projectFolder = 'my-project-from-Serverless-Land'
+    let rootNode: sinon.SinonSpiedInstance<AppBuilderRootNode>
+    let sandbox: sinon.SinonSandbox
+
+    beforeEach(async () => {
+        sandbox = sinon.createSandbox()
+        await fs.delete(path.join(workspaceFolder.uri.fsPath, projectFolder), { recursive: true })
+    })
+
+    afterEach(async () => {
+        await fs.delete(path.join(workspaceFolder.uri.fsPath, projectFolder), { recursive: true })
+        sandbox.restore()
+    })
+
+    describe(':) Path', async () => {
```

Comment:
This name isn't that clear. Should it include something with "serverlessland" in it?